app/shared/taginput/taginput.component.ts

```diff
@@ -107,4 +107,4 @@ export class TagInputComponent implements OnInit, OnChanges {
   ngOnInit() {
     this.tagPlaceholder = ' +' + this.role;
   }
-}
+}
```

Comment:
In this file you changed only spaces, but put more spaces in than necessary. please revert this file to its old version.